### PR TITLE
fix includes polyfill for ie

### LINKED ISSUE #13241: Array.prototype.includes() not supported by IE
When visiting beta through IE, the site is unusable. There are two instances of `includes` in the code base outside of the gulp file, and two solutions for fixing this.

1. Add a polyfill for `Array.prototype.includes()`
or
2. Refactor to remove `Array.prototype.includes()`

### PR DESCRIPTION
**Pull request by QuentinGibson**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13241

#### Description

Adds polyfills using babel-polyfill

### CODE REVIEW

**Review by Greenheart — COMMENTED** [review]
@QuentinGibson Thanks for the fix! :smile:

As far as I can tell, this is looking good. But I can't test in IE as I don't use Windows, so we'll need to wait for someone else to verify this.

File: client/rechallenge/transformers.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```
Comment (Bouncey) [inline client/rechallenge/transformers.js]: This should be imported into the entry point of the app which is `./client/index.js` 

This way it is available everywhere and we won't have to remember to import it everytime we want to use `includes()`

File: common/app/utils/blockNameify.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import 'babel-polyfill';
```
Comment (Bouncey) [inline common/app/utils/blockNameify.js]: As above

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: common/app/utils/blockNameify.js (hunk 2 above)
Comment (QuentinGibson) [inline common/app/utils/blockNameify.js]: Ok

**Pull request closed without merging**